[BD-4904] User delete (#10467)
braze-inc/braze-docs: _docs/_user_guide/data/unification/user_data/delete_users.md

---
nav_title: Deleting users
article_title: Deleting users
page_order: 4.2
toc_headers: h2
description: "TODO" 
---

# Deleting users

> Learn how to delete one or more user profiles directly through the Braze dashboard, without relying on calls to the `/users/delete` endpoint.

## Prerequisites

Before you can delete users, you'll need to be an admin or have **Delete User** permissions.

## About user deletion

User deletion lets you manage your database by removing profiles that are no longer needed, created in error, or required to be deleted for compliance (such as GDPR or CCPA).

| Consideration | Details |
|---------------|---------|
| Maximum size | You can delete up to 100 million user profiles in a bulk deletion. |
| Processing time | All bulk deletions require a 7-day waiting period plus the time it takes to process deletions. |
| Job limits | Only one bulk deletion can run at a time, which also includes the 7-day waiting period. |
{: .reset-td-br-1 .reset-td-br-2 role="presentation"}

{% alert tip %}
If you're on the FY26 pricing plan, be sure to regularly delete inactive user profiles to avoid additional charges.
{% endalert %}

## Deleting users

> ### Single user

Author: ```suggestion
### Individual
```

> In Braze, go to **Audience** > **Search Users**, then search for and select a user. If you're deleting a duplicate user profile, verify that you've selected the right one.

Author: ```suggestion
To delete an individual user from Braze, go to **Audience** > **Search Users**, then search for and select a user. If you're deleting a duplicate user profile, verify that you've selected the right one.
```

![ALT_TEXT]()

{% alert warning %}
Single-user deletions are permanent—profiles cannot be recovered after they're deleted.  
{% endalert %}

On their profile page, select **More Actions** > **Delete User**. Keep in mind, it may take a few minutes for the user to be fully deleted in Braze.

![ALT_TEXT]()

> ### Multiple users

Author: ```suggestion
### Bulk
```

{% endalert %}

## Canceling bulk deletions

You have 7 days to cancel bulk deletion requests. To cancel, go to **Audience** > **Manage Audience**, then select the **Delete Users** tab.

![ALT_TEXT]()

Next to a pending bulk deletion, select **Cancel**.

![ALT_TEXT]()

{% alert tip %}
When bulk user deletion is in progress, you can cancel it at any time. However, any users already deleted before the cancellation cannot be restored.
{% endalert %}

## Checking deletion status {#status}

To check the status of a deletion, use one of the following methods:

### Segment filters

When you delete a segment of users, a [segment filter]({{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/#filters) called **Pending Deletion** is automatically applied to your segment. You can use it to:

- See the exact set of users tied to a specific deletion run date.
- Exclude those users from campaigns so they don’t receive messages before removal.
- Export the list if you need it for compliance or record-keeping.

### Manage audience

{% alert note %}

> If your segment is dynamic, use the [Pending Deletion segment filter](#segment-filters) instead.

Author: hey john, i reached out on slack, but i'm not sure what you mean by this one 👍🏽

---
nav_title: Deleting users
article_title: Deleting users
page_order: 4.2
toc_headers: h2
description: "TODO" 
---

# Deleting users

> Learn how to delete one or more user profiles directly through the Braze dashboard.

{% alert important %}
This feature is currently in early access. Contact your customer success manager if you're interested in participating.
{% endalert %}

## Prerequisites

> Before you can delete users, you'll need to be an admin or have **Delete User** permissions.

Author: ```suggestion
To delete users, you'll need to be an admin or have **Delete User** permissions.
```

## About user deletion

User deletion lets you manage your database by removing profiles that are no longer needed, created in error, or required to be deleted for compliance (such as GDPR or CCPA).

| Consideration | Details |
|---------------|---------|
| Maximum size | You can delete up to 100 million user profiles when deleting a segment. |
| Waiting period | All segment deletions require a 7-day waiting period plus the time it takes to process deletions. |

> | Job limits | You can only segment can be deleted at a single time, which includes the 7-day waiting period. |

Author: ```suggestion
| Job limits | Only one segment can be deleted at a single time, which includes the 7-day waiting period. |
```

{: .reset-td-br-1 .reset-td-br-2 role="presentation"}

## Deleting users

> You can an [individual user](#delete-individual) or [segment of users](#delete-segment) through the Braze dashboard:

Author: ```suggestion
You can an [individual user](#delete-individual) or a [segment of users](#delete-segment) through the Braze dashboard:
```

### Deleting an individual {#delete-individual}

To delete an individual user from Braze, go to **Audience** > **Search Users**, then search for and select a user. If you're deleting a duplicate user profile, verify that you've selected the right one.

![The 'Search Users' page in Braze.]({% image_buster /assets/img/audience_management/duplicate_users/individual_merging/search_user.png %}){: style="max-width:75%;"}

{% alert warning %}
Single-user deletions are permanent—profiles cannot be recovered after they're deleted.  
{% endalert %}

> On their profile page, select **More Actions** > **Delete User**. Keep in mind, it may take a few minutes for the user to be fully deleted in Braze.

Author: ```suggestion
On their profile page, select select <i class="fa-solid fa-ellipsis"></i> **Show options** > **Delete User**. Keep in mind, it may take a few minutes for the user to be fully deleted in Braze.
```

### Deleting a segment {#delete-segment}

If you haven't already, [create a segment]({{site.baseurl}}/user_guide/engagement_tools/segments/creating_a_segment) containing the user profiles you want to delete. Be sure to include all user profiles if you're deleting duplicate users.

In Braze, go to **Audience** > **Manage Audience**, then select the **Delete Users** tab.

![The 'Delete Users' tab in the 'Manage Audience' section of the Braze dashboard.]({% image_buster /assets/img/audience_management/deleting_users/delete_users_tab.png %}){: style="max-width:85%;"}

Select **Delete users**, choose the segment you want to delete, then select **Next**.

![A pop-up window with a segment chosen for deletion.]({% image_buster /assets/img/audience_management/deleting_users/choose_segment_to_delete.png %}){: style="max-width:75%;"}

Type **DELETE** to confirm your request, then select **Delete users**.

![The confirmation page with 'DELETE' typed in the confirmation box.]({% image_buster /assets/img/audience_management/deleting_users/confirm_segment_delete.png %}){: style="max-width:75%;"}

> Your segment won't be deleted immediately. Instead, it will marked as pending deletion for 7 days. After this time, your segment will be deleted and we'll email you to let you know.

Author: ```suggestion
Your segment won't be deleted immediately. Instead, it will be marked as pending deletion for the next 7 days. After this time, your segment will be deleted and we'll email you to let you know.
```

> Your segment won't be deleted immediately. Instead, it will be marked as pending deletion for the next 7 days. After this time, your segment will be deleted and we'll email you to let you know.

Reviewer: The segment isn't marked.  Users in the segment will be marked with pending deletion.

> The users in this segment won't be deleted immediately. Instead, they'll be marked as pending deletion for the next 7 days. After this time, these users will be deleted and we'll email you to let you know.

Author: ```suggestion
The users in this segment won't be deleted immediately. Instead, they'll be marked as pending deletion for the next 7 days. After this time, these users will be deleted&#8212;we'll email you to let you know.
```

> The users in this segment won't be deleted immediately. Instead, they'll be marked as pending deletion for the next 7 days. After this time, these users will be deleted&#8212;we'll email you to let you know.

Author: ```suggestion
The users in this segment won't be deleted immediately. Instead, they'll be marked as pending deletion for the next 7 days. After this time, they'll be deleted and we'll email you to let you know.
```

{% alert tip %}

> To ensure that these exact users are deleted regardless of segment changes, a [segment filter]({{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/#filters) called **Pending Deletion** is automatically created. You can use this to check the status of pending deletions.

Reviewer: You don't use the filter to check the status.  You use the filter to see what users are marked for deletion.

{% endalert %}

## Canceling segment deletions {#cancel}

> You have 7 days to cancel segment deletions. To cancel, go to **Audience** > **Manage Audience**, then select the **Delete Users** tab.

Author: ```suggestion
You have 7 days to cancel pending segment deletions. To cancel, go to **Audience** > **Manage Audience**, then select the **Delete Users** tab.
```

![The 'Delete Users' tab in the 'Manage Audience' section of the Braze dashboard.]({% image_buster /assets/img/audience_management/deleting_users/delete_users_tab.png %}){: style="max-width:85%;"}

Next to a pending segment deletion, select <i class="fa-solid fa-eye"></i> to open the deletion record details.

![A pending segment deletion on the 'Delete Users' tab.]({% image_buster /assets/img/audience_management/deleting_users/pending_deletion.png %})

In the deletion record details, select **Cancel deletion**.

![The 'Deletion Record Details' window on the 'Delete Users' tab.]({% image_buster /assets/img/audience_management/deleting_users/deletion_record_details.png %}){: style="max-width:55%;"}

{% alert tip %}
When bulk user deletion is in progress, you can cancel it at any time. However, any users already deleted before the cancellation cannot be restored.
{% endalert %}

## Checking deletion status {#status}

> To check the status of a deletion, use one of the following methods:

Author: ```suggestion
You can check the status of a deletion using [segment filters](#segment-filters), the [manage audience](manage-audience) page, or [security event reports](#security-event-report).
```

### Segment filters

> When you delete a segment of users, a [segment filter]({{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/#filters) called **Pending Deletion** is automatically created. You can use it to:

Author: ```suggestion
When you request a segment of users to be deleted, a [segment filter]({{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/#filters) called **Pending Deletion** is automatically created. You can use it to:
```

- See the exact set of users tied to a specific deletion run date.
- Exclude those users from campaigns so they don’t receive messages before removal.
- Export the list if you need it for compliance or record-keeping.

### Manage audience

{% alert note %}

> If your segment is dynamic, use the [Pending Deletion segment filter](#segment-filters) instead.

Author: ```suggestion
To get the list of exact users that will be deleted, use the [Pending Deletion segment filter](#segment-filters) instead.
```

{% endalert %}

Go to **Audience** > **Manage Audience**, then select the **Delete Users** tab.

![The 'Delete Users' tab in the 'Manage Audience' section of the Braze dashboard.]({% image_buster /assets/img/audience_management/deleting_users/delete_users_tab.png %}){: style="max-width:85%;"}

On this page, you can find the following general information for all current and pending deletions:

| Field | Description |
|-------|-------------|
| Request Date | The date the request was originally made. Use it with the **Pending Deletion** filter to get the list of profiles pending deletion. |
| Requester | The user who initiated the deletion request. |

> | Segment Name | The segment that will be deleted. |

Reviewer: The segment name is actually the segment name that was initially used to mark users for pending deletion.  

To get the most accurate list, use the pending deletion filter.
